### ISSUE TITLE
1.6.1.1 Same old issues

### ISSUE THREAD

**Issue by uvxdev** [nicehash/NiceHashMiner-Archived#153]
More of the same. Wont benchmark Dagger, submits stales and crashes. Stays in memory. Re-launches Sgminer instead of ethminer on crash.

Last working version is still 1.5.1.0

Keep trying guys.....

**Comment by foxwarernd** [comment]
OS, GPU, ver. driver?

**Comment by d57heinz** [comment]
Win 10 pro
16.4.2. 390x + 2 290x
16gb ram amd8350 w/asus gaming 970 board. 

Just to confirm some of what he states.  When I have over clocked my rig too much when trying to find sweet spot it would crash the driver then you would have to stop the benchmark. But the sgminer stays running till you close it with the task manager.(only stays running on the two cards that I didn't oc too far. The crashed card load drops to 0)  I'm running a three gpu system that I tested this on. R9- 390x and 2 290x's.  Win10 with the latest drivers from Crimson suite. I have yet to test with the 15.7 as I don't want to go thru all that fiasco again if I don't have to. It's nice to stay current on drivers since mining isn't this computer sole use.   I think the odd behavior mostly is that it doesn't unload the program properly upon crash or overheat of vid card like one would expect.  This causes nicehashminer to load multiple instances of sgminer if you don't notice it still running and kill it manually.  Need any info or logs I can easily create this error.  Just overclock one card out of multi card system too far and bam.  Should do the trick.  I've tried this on farm puter but they are 6 card rigs and I didn't have the time to go thru which one is crashing it.  Seems X11 needs tweaked above and beyond what I set my gpu with on eth mining.  X11 gets so much hotter then eth.  Needs further trial and error on my part aswell. Sorry for such long post:)

Best Regards
d57heinz
Doug
